src/js/player.js

```diff
@@ -1292,7 +1292,13 @@ class Player extends Component {
    * @method play
    */
   play() {
-    this.techCall_('play');
+    // Only calls the tech's play if we already have a src loaded
+    if (this.player_.currentSrc()) {
+      this.techCall_('play');
+    } else {
+      this.autoplay(true);
```

Comment:
And I just realized that if we want it to autoplay after a source is given, we need to make sure that the `play` event is fired when this is called but it isn't going to right now, I don't think...